integration-cli/docker_cli_build_test.go

```diff
@@ -4591,3 +4591,20 @@ func TestBuildFromOfficialNames(t *testing.T) {
 	}
 	logDone("build - from official names")
 }
+
+// #4393
+func TestBuildVolumeFileExistsinContainer(t *testing.T) {
+	buildCmd := exec.Command(dockerBinary, "build", "-t", "docker-test-errcreatevolumewithfile", "-")
+	buildCmd.Stdin = strings.NewReader(`
+	FROM busybox
+	RUN touch /foo
+	VOLUME /foo
+	`)
+
+	out, _, err := runCommandWithOutput(buildCmd)
+	if err == nil || !strings.Contains(out, "file exists") {
+		t.Fatalf("exected build to fail when file exists in container at requested volume path")
```

Comment:
Should this read `expected build to fail...` instead of `exected build to fail...`?